code/computational_geometry/src/scanline_polygon_fill/HorizontalScanline.cpp

```diff
@@ -0,0 +1,107 @@
+#include<bits/stdc++.h>

```

Comment:
Include only relevant header files. Refer c++ style guide

Reply:
#1915 to be specific.